src/app/modules/recipe/components/recipe/overview/overview.component.ts

```diff
@@ -14,7 +14,11 @@ export class OverviewComponent implements OnInit {
 
     constructor(private store: Store) {
         store.select(AppState.getRecipes).subscribe((recipes) => {
-            this.recipes = recipes;
+            let copyRecipes = [...recipes];
+            let sortedRecipes = copyRecipes.sort((a: Recipe, b: Recipe) => {
```

Comment:
zou iets properder zijn als we zeggen:
`this.recipes = [...recipes];
this.recipes.sort ......
`